www/waterfall_view/src/module/waterfall.tpl.jade

```diff
@@ -3,6 +3,12 @@
         .spinner
             i.fa.fa-circle-o-notch.fa-spin.fa-2x
             p loading
+    p Tags: 
+        span(ng-show="w.all_tags.length > 0" ng-repeat="tag in w.all_tags")
```

Comment:
with metabuildbot, the list of tags is on only one line. I fear that for many tags you can't see all of them without scrolling. Can we fit them in several rows?

Reply:
Ok, I will send a PR to split them across rows (responsive)

Reply:
We discussed that already, but I wasn't sure about the best solution. We expect to have 20k+ tags in which case not even multiple lines will help to avoid scrolling, and I suggested that we may need to implement functionality to hide some of those tags (hiding would be done in `master.cfg`), only show the really relevant ones, and then add search-for-tags functionality.

Reply:
would the portnames be given tags in macports?

Reply:
I would like to be able to see just a single port in the waterfall view. But a prerequisite for that is that the tag with port name is hidden by default, else it would be impossible to find it anyway (even better if also searchable with some auto-complete functionality).